.gitlab-ci.yml

```diff
@@ -276,11 +272,11 @@ Image Tests:
   parallel:
     matrix:
       - RUNNER:
-          - aws/rhel-8.6-nightly-x86_64
-          - aws/rhel-8.6-nightly-aarch64
+          #- aws/rhel-8.6-nightly-x86_64
+          #- aws/rhel-8.6-nightly-aarch64
           # See https://github.com/osbuild/osbuild-composer/issues/1819 and related issues
-          - aws/rhel-9.0-nightly-x86_64
-          - aws/rhel-9.0-nightly-aarch64
+          #- aws/rhel-9.0-nightly-x86_64
+          #- aws/rhel-9.0-nightly-aarch64
         INTERNAL_NETWORK: ["true"]
```

Comment:
CI says "yaml invalid" I think he means this, maybe comment out the whole job for now if it's not going to have any runners?